### DISCUSSION TITLE
How to set indentation while using coc-format-select?

### DISCUSSION THREAD

**Discussion by LingyunStudio** [neoclide/coc.nvim#4668] [Q&A] [answered]
I am writting C program with nvim installed coc.

When i use c**oc-format-selected** to format my code, indentation becom 2 spaces, however i need 4 spaces.

How to set this? I failed to set it.
![image](https://github.com/neoclide/coc.nvim/assets/68700312/e68f8a6a-00e3-481d-bf5e-873415d22aef)

**Comment by fannheyward** [comment, marked as answer]
clangd embeds `clang-format` to format codes, you can set the indent with `.clang-format`. https://clangd.llvm.org/features#formatting
